dephell_pythons/_shell_utils.py

```diff
@@ -0,0 +1,25 @@
+import os
+from pathlib import Path
+
+
+# https://stackoverflow.com/a/377028/8704691
+def is_file(path: Path) -> bool:
+    try:
+        if not path.is_file():
+            return False
+    except PermissionError:
+        return False
```

Comment:
I don't think so. We're iterating over all binaries in PATH, and if some of them aren't available, it's not a problem. Let's just ignore them silently. Otherwise, we will get a bunch of unrelated warnings.

However, we can add `debug` here, you're right.

Reply:
Hmm, ok, seems reasonable.